### PR TITLE
Correct Missing Type Annotations #17

### PR DESCRIPTION
**Pull request by tcfranks**

### CODE REVIEW

**Comment by tcfranks** [comment]
I expect this isn't perfect....feedback pls....

File: adafruit_ble_file_transfer.py [hunk 1]
```diff
@@ -117,13 +123,13 @@ class ProtocolError(BaseException):
 class FileTransferClient:
     """Helper class to communicating with a File Transfer server"""
 
-    def __init__(self, service):
+    def __init__(self, service) -> None:
```
Comment (FoamyGuy) [inline adafruit_ble_file_transfer.py:126]: I think the argument here can be `Service`

File: adafruit_ble_file_transfer.py [hunk 2]
```diff
@@ -117,13 +123,13 @@ class ProtocolError(BaseException):
...
         self._service = service
 
         if service.version < 3:
             raise RuntimeError("Service on other device too old")
 
-    def _write(self, buffer):
+    def _write(self, buffer) -> None:
```
Comment (FoamyGuy) [inline adafruit_ble_file_transfer.py:132]: I think this one might be ReadableBuffer, but I'm not 100% sure, I'm still not confident I have the usages for them down correctly.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Thanks for working on this. The types that you have in are looking pretty good to me. I found a few functions without still that we can round out. 

I think we can type the `bind()` function here: https://github.com/tcfranks/Adafruit_CircuitPython_BLE_File_Transfer/blob/dca365e252cb829e44fe975698a9ed6321a40e72/adafruit_ble_file_transfer.py#L68 with `Service` and PacketBuffer` types for argument and returns.

**Comment by tcfranks** [comment]
Ready for 2nd review, I think.

File: adafruit_ble_file_transfer.py (hunk 2 above)
Comment (tekktrik) [reply adafruit_ble_file_transfer.py]: You got it!

**Pull request merged**